cmd/cosign/cli/verify/verify_blob.go

```diff
@@ -73,16 +74,16 @@ func (c *VerifyBlobCmd) Exec(ctx context.Context, blobRef string) error {
 	opts := make([]static.Option, 0)
 
 	// Require a certificate/key OR a local bundle file that has the cert.
-	if options.NOf(c.KeyRef, c.CertRef, c.Sk, c.BundlePath, c.RFC3161TimestampPath) == 0 {
-		return fmt.Errorf("please provide a cert to verify against via --certificate or a bundle via --bundle or --rfc3161-timestamp-bundle")
```

Comment:
This changes the functionality forcing the usage of a bundle path instead of making it optional to passing a rfc3161 timestamp.

Reply:
This is intentional. I don't think we should conflate the 3161 timestamp with the on-disk bundle.

To verify an artifact, you need:
* A signature, which can be provided in the "--bundle" or via a "--sig" flag
* A verification key, provided by "--certificate", "--sk", or "--bundle"

I would prefer to not change the on-disk bundle format and treat the timestamp as separate. I think it's a little confusing to bundle the artifact certificate with the timestamp, since those are unrelated. I would prefer that each element be provided separately.

So tl;dr, you would either:
* provide a --certificate flag, --signature flag, and --3161timestamp
* provide a --bundle flag and --3161timestamp

Reply:
Another reason I wanted to do this is that `--bundle` == `LocalSignedPayload`, and I didn't want to make changes to that struct. To me, it seems only like an optimization to include the 3161 timestamp in `LocalSignedPayload`, and I think simply tacking on a timestamp flag when you want to use the timestamp is cleaner and easier to explain.

Another option would be a different struct for storing the ts locally, like `Local3161Bundle`, but I think this is confusing to add another bundle format especially as we're working on another bundle format for the specs.

Reply:
👍🏻 It sounds good to me.